### DISCUSSION TITLE
WALC 0.2.3

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#209] [General]
Bug fixes:
- Fix dashboard icon not showing
- Fix whatsapp integration
- Disabled notification grouping (temporarily) since it causes problem when replying

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.2.3'>WALC 0.2.3</a>.</em>

**Comment by albakhtari** [comment]
Updated the AUR package.

**Comment by lashkevi** [comment]
How could I open settings? There is no menu at all.

**Reply by daffamumtaz2361** [reply]
There should be a WALC icon that you can click next to status button

**Reply by albakhtari** [reply]
![image](https://user-images.githubusercontent.com/79008923/200157175-b674697e-7d10-45a2-a6b5-a331650f58f2.png)
![image](https://user-images.githubusercontent.com/79008923/200157196-6d815d65-4c73-48c0-87f8-d5d36513a839.png)
(To show the menu items, pres the `alt` button)

**Comment by lashkevi** [comment]
No icon. Ctrl+D doesn't work as well.
![Screenshot_20221106_115442](https://user-images.githubusercontent.com/13115223/200162436-ed9e4015-ec0e-475a-bdbf-4a81ada155de.png)

**Reply by albakhtari** [reply]
You need to update WALC.
Your icon is still the old one.

You can download the AppImage from [here](https://github.com/WAClient/WALC/releases/download/v0.2.3/WALC-0.2.3.AppImage).

Or if you run an Arch based distro, you can install it from the AUR via:
```
yay -S walc
```

**Comment by lashkevi** [comment]
After several days of work WALC is again identified as Chrome version under 60 and stops working. Something is wrong with useargent, I guess.

**Comment by albakhtari** [comment]
Integration is broken again :(
Maybe bumping the dependencies would help?

**Reply by daffamumtaz2361** [reply]
Yes it has been for a few days now. But whatsapp-web.js just released a fix for it yesterday.

Gonna update soon with a few bug fixes too, like enabling notification grouping again
